app_dart/lib/src/service/datastore.dart

```diff
@@ -166,6 +166,20 @@ class DatastoreService {
     }
   }
 
+  // Queries for recent tasks without considering branches.
+  Stream<FullTask> queryRecentTasksNoBranch(
```

Comment:
A new test in datastore_test for this new method?

Reply:
Added.